Example/PrebidDemo/PrebidDemo.xcodeproj/project.pbxproj

```diff
@@ -813,7 +846,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = PrebidDemoSwift/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
yeah, it was discussed with @ppuviarasu and @littleV that we should upgrade to iOS 10. And since for iOS 9 as per Apple here, https://developer.apple.com/support/app-store/ users percentage is almost negligible.